the-shore/patterns/home.php

```diff
@@ -0,0 +1,205 @@
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_image_7.jpg" alt="<?php echo __('', 'the-shore');?>"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"66.66%"} -->
+<div class="wp-block-column" style="flex-basis:66.66%"><!-- wp:image {"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_image_8.jpg" alt="<?php echo __('', 'the-shore');?>"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:image {"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_image_9.jpg" alt="<?php echo __('', 'the-shore');?>"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:spacer {"height":"50px"} -->
+<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"metadata":{"name":"Banner with CTA"},"style":{"spacing":{"blockGap":"2.8rem"},"elements":{"link":{"color":{"text":"var:preset|color|base"}}}},"backgroundColor":"primary","textColor":"base","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-base-color has-primary-background-color has-text-color has-background has-link-color"><!-- wp:spacer {"height":"50px"} -->
+<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:image {"width":"380px","sizeSlug":"full","linkDestination":"none","align":"center"} -->
+<figure class="wp-block-image aligncenter size-full is-resized"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/line_light.png" alt="<?php echo __('', 'the-shore');?>" style="width:380px"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:paragraph {"align":"center","style":{"typography":{"textTransform":"uppercase","letterSpacing":"0.1em","fontStyle":"normal","fontWeight":"500"}},"fontSize":"x-small","fontFamily":"commissioner"} -->
+<p class="has-text-align-center has-commissioner-font-family has-x-small-font-size" style="font-style:normal;font-weight:500;letter-spacing:0.1em;text-transform:uppercase"><?php echo __('The Shore Spa®', 'the-shore');?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"textAlign":"center","align":"wide","style":{"typography":{"fontStyle":"normal","fontWeight":"300"},"elements":{"link":{"color":{"text":"var:preset|color|base"}}}},"textColor":"base"} -->
+<h2 class="wp-block-heading alignwide has-text-align-center has-base-color has-text-color has-link-color" style="font-style:normal;font-weight:300"><?php echo __('Experience a holistic approach to wellness in our tranquil spa setting, where every detail is crafted for your ultimate comfort.', 'the-shore');?></h2>
```

Comment:
Same as above, "The Shore Spa" should be the `<h2>` and `Experience a holistic approach to wellness in our tranquil spa setting, where every detail is crafted for your ultimate comfort.` should be the `<p>`